package/klish/klish.conf

```diff
@@ -0,0 +1,21 @@
+# Template for config file /etc/klish/klish.conf. It's used by klish utility.
+
+# The klishd uses UNIX domain socket to receive connections. It will create an
+# filesystem entry to allow clients to find connection point. By default klish
+# client utility will connect to /tmp/klish-unix-socket.
+#UnixSocketPath=/tmp/klish-unix-socket
+
+# The klish can use external pager for non-interactive commands. By default it
+# will execute "/usr/bin/less -I -F -e -X -K -d -r -S" process as a pager.
+Pager="/usr/bin/less -I -F -e -X -K -d -r -S"
```

Comment:
Should we leave it commented out, since it is the default?

Reply:
Ah, misleading comment. I've changed to -r and added -S. I'll update the comment.

Reply:
Added another commit to clean up/out the misleading comments (and move the klishd socket file to /run).